python/cuml/__init__.py

```diff
@@ -46,6 +46,8 @@
 from cuml.solvers.sgd import SGD
 from cuml.solvers.qn import QN
 
+from cuml.tsa.holtwinters import HoltWinters
```

Comment:
One thing I just realized is that we should probably rename the class to `ExponentialSmoothing` to match statsmodels.

Reply:
Yeah, to exactly match that is right. Practitioners would recognize Holt Winters as a synonym for exponential smoothing, I don't think we would fool many people, but we want to fool 0 people.

Reply:
Walmart calls this Holt-Winters, that's why we called it that...

Reply:
No problem. While it's a trivial change to make, and while we are following the statsmodels API, I just want to make sure we're being as `drop-in replacement` as possible.